core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -1243,6 +1289,40 @@ abstract class Bundle(implicit compileOptions: CompileOptions) extends Record {
     VectorMap(nameMap.toSeq.sortWith { case ((an, a), (bn, b)) => (a._id > b._id) || ((a eq b) && (an > bn)) }: _*)
   }
 
+  /* This is for checking compatibility between using bundle plugin vs not using it
+   */
+  def oldElementsNoChecks: SeqMap[String, Data] = {
+    val nameMap = LinkedHashMap[String, Data]()
+    for (m <- getPublicFields(classOf[Bundle])) {
+      getBundleField(m) match {
+        case Some(d: Data) =>
+          if (nameMap contains m.getName) {} else {
+            nameMap(m.getName) = d
+          }
+        case None =>
+          if (!ignoreSeq) {
+            m.invoke(this) match {
+              case s: scala.collection.Seq[Any] if s.nonEmpty =>
+                s.head match {
+                  // Ignore empty Seq()
+                  case d: Data =>
+                    throwException(
+                      "Public Seq members cannot be used to define Bundle elements " +
+                        s"(found public Seq member '${m.getName}'). " +
+                        "Either use a Vec if all elements are of the same type, or MixedVec if the elements " +
+                        "are of different types. If this Seq member is not intended to construct RTL, mix in the trait " +
+                        "IgnoreSeqInBundle."
+                    )
+                  case _ => // don't care about non-Data Seq
+                }
+              case _ => // not a Seq
+            }
+          }
+      }
+    }
+    VectorMap(nameMap.toSeq.sortWith { case ((an, a), (bn, b)) => (a._id > b._id) || ((a eq b) && (an > bn)) }: _*)
+  }
+
```

Comment:
This was useful for development but this needs to be private if we keep it.

Reply:
Making this private